NuclearData/mgNeutronData/baseMgNeutron/Tests/baseMgNeutronDatabase_iTest.f90

```diff
@@ -207,6 +210,8 @@ subroutine testBaseMgNeutronDatabaseWithP1()
     @assertEqual(4, database % nGroups())
 
     ! Test getting Transport XS
+    ! Associate pointer to pass tests in debug mode
+    p % pRNG => pRNG
```

Comment:
Do we know why it was unhappy?
Was it because we provide the pointer later to a function with a non-pointer dummy argument (if it is the case thank you GCC!)

Reply:
I don't think so. The error I was getting was because the particle gets passed to a few functions (something like ``mgDatabase % getTrackingXS`` -> ``mgDatabase % getTotalXS`` -> ``mgMaterial % getTotalXS``) where p % pRNG is an argument, and the problem was that this is a pointer and it was unassigned.